book/src/prefabs.md

```diff
@@ -0,0 +1,50 @@
+# Prefabs
+
+Premade fabrications, or prefabs, are templates that specify components to attach to an entity. For example, imagine a monster entity has the following components:
+
+* Position
+* Velocity
+* Texture
+* Health points
+* Attack damage
+
+It is certainly possible to define the values for each monster in code. However, if the components are initialized in code, then the executable needs to be recompiled whenever the component values are changed. Waiting a number of minutes to every time a small change is made is both inefficient and frustrating.
+
+In data-oriented design (DoD), instantiating monsters is logic and is part of the executable, but the values to use for the components is data. The executable can instantiate any monster using the data for the monsters read from a *prefab* file like the following:
```

Comment:
Unless the abbreviation "DoD" here is used subsequently in the text I believe it's unnecessary.